README.md

```diff
@@ -16,58 +21,71 @@ The iExec Secret Management Service (SMS) stores user secrets and provisions the
 
 ## Configuration
 
-The iExec Secret Management Service is available as an OCI image on [Docker Hub](https://hub.docker.com/r/iexechub/iexec-sms/tags).
+The _iExec Secret Management Service_ is available as an OCI image on [Docker Hub](https://hub.docker.com/r/iexechub/iexec-sms/tags).
+
+A single _iExec Secret Management Service_ instance supports a single TEE framework.
+To support both Scone and Gramine TEE tasks, two instances of _iExec SMS_ must be configured.
 
-To run properly, the iExec Secret Management Service requires:
+To run properly, the _iExec Secret Management Service_ requires:
 * A blockchain node. iExec smart contracts must be deployed on the blockchain network.
-* A Scontain Configuration and Attestation Service (CAS) to provision secrets to remote enclaves.
-* Valid OCI images configurations for pre-compute and post-compute stages of TEE tasks executions.
-* A valid OCI image configuration of a Scontain Local Attestation Service (LAS).
-  This service will be deployed by an iExec Worker to compute TEE tasks.
+* Valid OCI images configurations for pre-compute and post-compute stages of TEE tasks executions. Exposed images depend on the type of TEE framework supported.
+* A Secret Provisioner instance, in charge of provisioning secrets to remote enclaves. Each TEE framework requires its own type of Secret Provisioner.
+    * for Scone TEE tasks:
+        * a Scontain _Configuration and Attestation Service_ (CAS).
+        * a valid OCI image configuration of a Scontain _Local Attestation Service_ (LAS). This service will be deployed by an iExec Worker to compute TEE tasks.
+    * for Gramine TEE tasks:
+        * an _iExec SPS_ instance.
```

Comment:
For compliance with CAS, LAS, SMS, do we have the words behind the `SPS` acronym ?

Reply:
Better in full yes, here it is https://github.com/iExecBlockchainComputing/iexec-sms/pull/155/commits/31db9d00a74c9bd5eb1a82d8bf4a19b54aceac11 :pray: